app/models/merchant.rb

```diff
@@ -1,4 +1,5 @@
 class Merchant < ApplicationRecord
+  validates :name, presence: true
```

Comment:
Do you have a model test for this? Should be something along the lines of:

  it { should validate_presence_of(:name) }

Reply:
I do now, It will be included with my create merchant test branch